opendebates/views.py

```diff
@@ -330,6 +330,17 @@ def questions(request):
 class OpenDebatesRegistrationView(RegistrationView):
 
     form_class = OpenDebatesRegistrationForm
+    next = None
+
+    def get(self, request):
+        self.next = request.GET.get('next', None)
+        return super(OpenDebatesRegistrationView, self).get(request)
+
+    def get_context_data(self, *args, **kwargs):
+        data = super(OpenDebatesRegistrationView, self).get_context_data(*args, **kwargs)
+        if self.next:
+            data['next'] = self.next
```

Comment:
I think this will fail on a registration POST when there's no `next` attribute on `self`.